pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -959,6 +960,9 @@ private List<QuestionRecord> createQuestionRecordsForActivity(ActivityDef defini
                             allQuestions.add(questionBlock.getQuestion());
                         }
                     }
+                } else if (formBlock.getBlockType() == BlockType.TABULAR) {
+                    TabularBlockDef tabularBlock = (TabularBlockDef) formBlock;
+                    allQuestions.addAll(tabularBlock.getQuestions().collect(Collectors.toList()));
```

Comment:
DataExporter stuff looks ok to me.
Please make sure you run DataExporterCli for elastic and csv export and make sure questions in the new block & responses/answers  are handled and exported, particularly elastic.

Reply:
Got it, will do after merging